apps/fipsinstall.c

```diff
@@ -50,7 +51,9 @@ const OPTIONS fipsinstall_options[] = {
     {"provider_name", OPT_PROV_NAME, 's', "FIPS provider name"},
     {"section_name", OPT_SECTION_NAME, 's',
      "FIPS Provider config section name (optional)"},
-     {"no_conditional_errors", OPT_NO_CONDITIONAL_ERRORS, '-',
+    {"no_pkcs1_v1_5_padding", OPT_NO_PKCS1V15_PADDING, '-',
```

Comment:
SO at some point this will become default to be disabled. So I dont think this inverse 'no' option is very forward thinking. Make it enable/disable flag.. If you dont specify the option the default is that it is disabled (for now). Or should we just disable it by default, and document it... I get that you are trying not to break tests.

Reply:
Consistency?  There are **exactly** **two** instances of the enable/disable paradigm in the commands:
* `enable_pha`
* `disable_confirm`
Neither has the alternative option.

I got the description of the default wrong, it defaults to allowed.